targets/ESP32/_nanoCLR/nanoFramework.Espnow/nanoFramework_espnow_native_nanoFramework_Espnow_EspNowController.cpp

```
@@ -0,0 +1,206 @@
+// Copyright (c) .NET Foundation and Contributors
+// See LICENSE file in the project root for full license information.
+//
+
+#include "nanoFramework_espnow_native.h"
+
+EspNowDataSentEventData Library_nanoFramework_EspNow_native_nanoFramework_EspNow_EspNowController::dataSentEventData;
```

Clang (14.0.6)

[warning] variable 'dataSentEventData' is non-const and globally accessible, consider making it const

(cppcoreguidelines-avoid-non-const-global-variables)